Remove bower and replace it with yarn

bassjobsen opened the pull request:
according: https://bower.io/blog/2017/how-to-migrate-away-from-bower/

a1300 commented:
__REVIEW__
1. Do we still need the [bower.json](https://github.com/AschPlatform/asch/blob/master/public/bower.json) file?
    * Can it be removed?
2. Could you also please update:
   *  [asch/README.md](https://github.com/AschPlatform/asch/blob/master/README.md) #190 
   * [asch/.gitignore](https://github.com/AschPlatform/asch/blob/8ac58ac97bfac6bcc66165eda3dd85653390c79b/.gitignore) 
   * [asch/docs/asch_dev_windows_install.md](https://github.com/AschPlatform/asch/blob/55bfb3b43db1d8e8e0b8d1884aa4b3f911a1a849/docs/asch_dev_windows_install.md)
   *  [asch/docs/dapp_docs/dapp_dev_tutorials.md](https://github.com/AschPlatform/asch/blob/4f4da2064ddb47c2ee91df434bdbfae17b361315/docs/dapp_docs/dapp_dev_tutorials.md)
   * and maybe the [asch/docs/dapp_docs/1_hello_en.md](https://github.com/AschPlatform/asch/blob/a0b3740107128d3f1efb2c0a60a394d0b34d3144/docs/dapp_docs/1_hello_en.md) file, the portion where the web-wallet is installed.
3. Please commit the `yarn.lock` file
    * According to [stackoverflow](https://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for) and [yarn/blog](https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/) it is best practise to commit the `yarn.lock` file.
3. Could you squash all of your commits into one file
    * Like ```git reset --soft HEAD~3 && git commit -m "Squash message"```

Thanks for your effort! :+1: 
Best regards
a1300

bassjobsen commented:
@a1300 thanks for your review and sugestions. You was right about both the `bower.json` and `yarn.lock`. I missed the `bower install` in the `package.json` `pre-install`, i've removed it. I've also changed the files you suggested.
